packages/server/src/controllers/Controller.js

```diff
@@ -533,3 +475,54 @@ export class Controller {
 EventEmitter.mixin(Controller.prototype)
 
 const inheritanceMap = new WeakMap()
+
+const setupHandlerFromObject = ({
+  handler,
+  action,
+  authorize,
+  parameters,
+  returns,
+  scope,
+  transacted
+}) => {
+  handler.authorize = authorize
+  handler.transacted = transacted
+
+  if (action) {
+    const [verb, path] = asArray(action)
+    handler.verb = verb
+    handler.path = path
+  }
+
+  if (parameters) {
+    const [_parameters, options] = parameters
+    const hasOptions = isArray(_parameters)
+    handler.parameters = hasOptions ? _parameters : parameters
```

Comment:
Can't you simply do `options  ? _parameters : parameters` now, without the need for  `hasOptions`?

Reply:
We are handling two cases here: `parameters: [parameters, options]` and `parameters: [parameterA, parameterB, etc]`. In both of this cases `options` can contain a value and therefore be truthy.